src/redis-benchmark.c

```diff
@@ -1854,19 +1872,19 @@ int main(int argc, char **argv) {
         }
 
         if (test_is_selected("set")) {
-            len = redisFormatCommand(&cmd,"SET key%s:__rand_int__ %s",tag,data);
+            len = redisFormatCommand(&cmd,"SET key%s:__rand_int__%s %s",tag,suffix,data);
```

Comment:
When computing the suffix length, we should take into account the `tag` size as well. (It's only used in cluster mode.)